ci/docker/image-builder/src/main/kotlin/ru/avito/image_builder/internal/command/EmulatorImageBuilder.kt

```diff
@@ -0,0 +1,121 @@
+package ru.avito.image_builder.internal.command
+
+import ru.avito.image_builder.internal.docker.CliDocker
+import ru.avito.image_builder.internal.docker.ContainerId
+import ru.avito.image_builder.internal.docker.Image
+import ru.avito.image_builder.internal.docker.ImageId
+import java.io.File
+import java.util.logging.Logger
+
+internal class EmulatorImageBuilder(
```

Comment:
All meaningful logic for emulators here.
It's mostly a copy from `docker-in-docker` behavior.